panacea/config/config.exs

```diff
@@ -19,7 +19,7 @@ config :logger, :console,
   metadata: [:request_id]
 
 config :panacea, :asclepius,
-  uri: URI.parse("http://asclepius:5000"),
+  uri: URI.parse("http://localhost:5000"),
```

Comment:
Don't think this needs to be overridden though because asclepius is mocked in test. When we add integration tests they'll run panacea in prod so the http adapter gets tested properly too